content/reference/sql/benchmarks/latency.md

```diff
@@ -60,4 +62,42 @@ The Dolt version is `0.40.29`.
 | Overall Mean Multiple | 8.2 |
 |-----------------------|-----|
 <!-- END_LATENCY_RESULTS_TABLE -->
+
+### New Format (`__DOLT__`)
+
+Below are the results of running `sysbench` MySQL tests against Dolt
+SQL Server for the most recent release of Dolt in the [new 
+storage format](https://www.dolthub.com/blog/2022-08-12-new-format-migraiton/).
```

Comment:
```suggestion
storage format](https://www.dolthub.com/blog/2022-08-12-new-format-migration/).
```

Reply:
the link is correct actually.... taking my shame

Reply:
Unfortunately, the mispelled one is aktually the link :-)

Try: https://www.dolthub.com/blog/2022-08-12-new-format-migration/